content/chef_compliance_phase.md

```diff
@@ -0,0 +1,431 @@
++++
+title = "Compliance Phase"
+draft = false
+
+aliases = ["/chef_compliance_phase.html"]
+
+[menu]
+  [menu.infra]
+    title = "Compliance Phase"
+    identifier = "chef_infra/features/chef_compliance_phase.md Compliance Phase"
+    parent = "chef_infra/features"
+    weight = 80
++++
+
+[\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/chef_compliance_phase.md)
+
+## Overview
+
+Compliance Phase enables Chef Infra to run Chef InSpec profiles as part of a client run, by downloading profiles from sources like Chef Automate, Chef Supermarket, or GitHub, and reporting the results to Chef Automate or saving the results locally. Compliance Phase has been implemented to be mostly compatible with [audit cookbook](https://github.com/chef-cookbooks/audit), so that users of the audit cookbook can remove it from the node's runlist and get the same behavior.
+
+## Usage
+
+### Configuration
+
+Compliance Phase needs to be configured for each node where the `chef-client` runs. All node-specific configuration is done via Chef Infra attributes. Specifying one or more Chef InSpec profiles is the minimum configuration necessary to enable Compliance Phase in a Chef Infra client run. With no further configuration, this will cause a Chef Infra client run to execute the Chef InSpec profiles and write the results to disk using the default `json-file` reporter.
+
+#### Profiles
+
+Chef InSpec profiles are specified using the `node['audit']['profiles']` attribute.
+
+##### Automate
+
+```ruby
+default['audit']['profiles']['linux-baseline'] = {
+  'compliance': 'user/linux-baseline',
+  'version': '2.1.0'
+}
+```
+
+Note: Fetching InSpec profiles from a Chef Automate server will require additional setup for authorization and authentication.
+
+##### Supermarket
+
+```ruby
+default['audit']['profiles']['ssh'] = {
+  'supermarket': 'hardening/ssh-hardening'
+}
+```
+
+##### Local paths
+
+```ruby
+default['audit']['profiles']['brewinc/win2012_audit'] = {
+  'path': 'E:/profiles/win2012_audit'
+}
+```
+
+##### Git
+
+```ruby
+default['audit']['profiles']['ssl'] = {
+  'git': 'https://github.com/dev-sec/ssl-benchmark.git'
+}
+```
+
+##### HTTP
+
+```ruby
+default['audit']['profiles']['ssh2'] = {
+  'url': 'https://github.com/dev-sec/tests-ssh-hardening/archive/master.zip'
+}
+```
+
```

Comment:
This will put all this in a [tabs container](https://docs.chef.io/style_guide/#foundation-tabs-container)

```suggestion
#### Profiles

You can fetch [Chef InSpec profiles]({{< relref "inspec/profiles/" >}}) from several different locations using the `node['audit']['profiles']` attribute.

For example:

{{< foundation_tabs tabs-id="compliance-phase-profile-panel" >}}
  {{< foundation_tab active="true" panel-link="automate-panel" tab-text="Automate">}}
  {{< foundation_tab panel-link="supermarket-panel" tab-text="Supermarket" >}}
  {{< foundation_tab panel-link="local-path-panel" tab-text="local path" >}}
  {{< foundation_tab panel-link="git-panel" tab-text="Git" >}}
  {{< foundation_tab panel-link="http-panel" tab-text="HTTP" >}}
{{< /foundation_tabs >}}

{{< foundation_tabs_panels tabs-id="compliance-phase-profile-panel" >}}
  {{< foundation_tabs_panel active="true" panel-id="automate-panel" >}}
  ```ruby
  default['audit']['profiles']['linux-baseline'] = {
    'compliance': 'user/linux-baseline',
    'version': '2.1.0'
  }
  ```

  **Note**: Fetching InSpec profiles from a Chef Automate server will require additional setup for authorization and authentication.

  {{< /foundation_tabs_panel >}}

  {{< foundation_tabs_panel panel-id="supermarket-panel" >}}
  ```ruby
  default['audit']['profiles']['ssh'] = {
    'supermarket': 'hardening/ssh-hardening'
  }
  ```

  {{< /foundation_tabs_panel >}}
  {{< foundation_tabs_panel panel-id="local-path-panel" >}}
  ```ruby
  default['audit']['profiles']['brewinc/win2012_audit'] = {
    'path': 'E:/profiles/win2012_audit'
  }
  ```
  {{< /foundation_tabs_panel >}}
  {{< foundation_tabs_panel panel-id="git-panel" >}}
  ```ruby
  default['audit']['profiles']['ssl'] = {
    'git': 'https://github.com/dev-sec/ssl-benchmark.git'
  }
  ```
  {{< /foundation_tabs_panel >}}
  {{< foundation_tabs_panel panel-id="http-panel" >}}
  ```ruby
  default['audit']['profiles']['ssh2'] = {
    'url': 'https://github.com/dev-sec/tests-ssh-hardening/archive/master.zip'
  }
  ```
  {{< /foundation_tabs_panel >}}
{{< /foundation_tabs_panels >}}

```